src/orion/plotting/_core.py

```diff
@@ -0,0 +1,85 @@
+import orion.plotting.backend_plotly as backend
+
+
+def regret(experiment, order_by='suggested', verbose_hover=False, **kwargs):
+    """
+    Make a plot to visualize the performance of the hyper-optimization process.
+
+    The x-axis contain the trials and the y-axis their respective best performance.
+
+    Parameters
+    ----------
+        experiment: ExperimentClient, Experiment or ExperimentView
```

Comment:
Unless numpy docstring official format changed recently I think the body of Parameters should be only 1 indent.

Reply:
You're right. I don't know why this one is indented